src/test/java/api/update/UpdateUserPositiveTest.java

```diff
@@ -18,15 +18,15 @@
 import java.util.Collections;
 
 import static org.realworld.api.Constants.EMAIL_SUFFIX;
-import static org.realworld.api.Constants.USER_NAME_PREFIX;
+import static org.realworld.api.Constants.USER_NAME_PREFIX_API;
 
 public class UpdateUserPositiveTest {
 
     private ApiService authorizedApiService;
 
     @BeforeMethod(alwaysRun = true)
     public void signIn() {
-        String userName = USER_NAME_PREFIX + System.nanoTime();
+        String userName = USER_NAME_PREFIX_API + System.nanoTime();
```

Comment:
**minor**: because sometime you add milliseconds to the nano time, consider to use 
```
UUID randomUUID = UUID.randomUUID();
randomUUID.toString()
```
https://xperti.io/blogs/generate-random-string-in-java/

Thus, you always gonna get a new unique name, with no necessity to think about those additional milliseconds

Reply:
done